views/term_table.erb

```diff
@@ -112,12 +112,12 @@
 
                 <span class="person-card-filter">
                     <label for="filter-input" class="fa fa-search"></label>
-                    <input type="text" class="js-filter-input" id="filter-input" placeholder="Search by name, place, etc…">
+                    <input type="text" class="js-filter-input" id="filter-input" placeholder="Search by name, place, etc…" data-ga-track-change="Filter field">
```

Comment:
Maybe "filter term politician grid" or something might be a more useful label for the event?

(Just thinking, when we're looking at the reports in Google Analytics, it'll be much more obvious what and where the "filter term politicians" event came from, than the vague sounding "filter field")